main/js/api.js

```diff
@@ -61,14 +66,26 @@ async function startUpdate() {
     const status = document.getElementById('status').value;
     const candidateIds = document.getElementById('candidateIds').value.split(',');
     const apiInstance = document.getElementById('apiInstance').value;
+    const requestTotal = candidateIds.length;
+    let requestCount = 0;
+    document.getElementById('output-progress').style.visibility = 'visible';
+    document.getElementById('stop').disabled = false;
 
     try {
         const authToken = await getAuthToken(accountId, secretKey, apiInstance);
+        abortController = new AbortController();
+        const signal = abortController.signal;
```

Comment:
Instantiate in the API call function to ensure a fresh instance is created each time the process starts.